Examples/Physics_applications/laser_ion/README.rst

```diff
@@ -30,7 +24,12 @@ This example can be run **either** as:
 * **Python** script: ``mpiexec -n 2 python3 PICMI_inputs_2d.py`` or
 * WarpX **executable** using an input file: ``mpiexec -n 2 warpx.2d inputs_2d``
 
-For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+.. tip::
+
+   For `MPI-parallel <https://www.mpi-forum.org>`__ runs on computing clusters, change the prefix to ``mpiexec -n <no. of MPI ranks> ...`` or ``srun -n <no. of MPI ranks> ...``, depending on the system and number of MPI ranks you want to allocate.
+
+   The input option ``warpx_numprocs`` / ``warpx.numprocs`` needs to be adjusted for parallel :ref:`domain decomposition <usage_domain_decomposition>`, to match the number of MPI ranks used.
```

Comment:
```suggestion
   The input option ``warpx_numprocs`` (or, alternatively, the two options ``amr.max_grid_size`` and ``amr.blocking_factor``) needs to be adjusted for parallel :ref:`domain decomposition <usage_domain_decomposition>`, to match the number of MPI ranks used.
```

Reply:
The two mentions are slightly different: one Python, one inputs file :)

The blocking factor and details are explained in better detail in the link.

Reply:
@lucafedeli88 , I would merge for now - but we can find a more general text we use on all examples :)